api/cds.proto

```diff
@@ -319,4 +319,12 @@ message Cluster {
 
   // See base.TransportSocket description.
   TransportSocket transport_socket = 24;
+
+  // The Metadata field can be used to provide additional information
+  // about the cluster. It can be used for stats, logging, and varying filter
+  // behavior. Fields should use namespace notation to denote which entity
+  // within Envoy will need the information. For instance, if the metadata
+  // is intended for the Router filter, the filter name should be specified as
+  // *envoy.router*.
```

Comment:
Can you add a `[#not-implemented-hide:]` or `[#not-implemented-warn:]` annotation for protodoc?

Reply:
Of course, if you plan on implementing it immediately, this isn't needed.